modules/datadog/variables.tf

```diff
@@ -63,3 +63,8 @@ variable "datadog_agent_version_fargate" {
   type        = string
   default     = "7.54.0"
 }
+
+variable "product_name" {
+  description = "Name of the product"
+  type        = string
+}
```

Comment:
Without default this is a required var opinions on a possible default? used in [modules/datadog/main.tfR121

Reply:
what about _example_ ?

Reply:
But I prefer not setting it so that the user is required to set something meaningful.